### ISSUE TITLE
Inconsistent use of volume fraction in vesicle model

### ISSUE THREAD

**Issue by gnsmith** [SasView/sasview#2794]
**Describe the bug**
I will include screenshots from a selected simulation to illustrate the bug.

The "vesicle" model (category: sphere) includes a term for volume fraction of shell material. As far as I can tell from the documentation (https://www.sasview.org/docs/user/models/vesicle.html), this is done for volume normalisation to reduce the scattering to that from the shell only.
![SasView_pq](https://github.com/SasView/sasview/assets/46593797/a7960079-7185-4d3a-9bc9-1312c74164cb)
When adding a structure factor, this volume fraction is then taken as the structure factor volume fraction.
![SasView_pqsq](https://github.com/SasView/sasview/assets/46593797/6d236987-9505-4c5f-a77b-ccea11e9a297)
However, while they have the same names, they are conceptually different. The volume fraction (for form factor), as described in the documentation, is the fraction of material added to the system. The volume fraction (for structure factor) needs to be the volume of the shell and the core, as the vesicles will fill up the entire volume (4/3*pi*R_tot^3).

Using the values I put in this simulation and taking the sphere-equivalent volume of the shell, this means that the real volume fraction for the structure factor should be 1.42x that inputted. This will obviously differ for different combinations of radius and thickness.

**To Reproduce**
1. Change Fit panel to the correct model, Category:Sphere and Model name:vesicle.
2. Simulate scattering without a structure factor.
3. Add in a structure factor model. The volfraction will not change.

**Expected behavior**
I can see two possibilities.
- There can be be two volume fractions: one for volume normalisation and one for the structure factor.
- One of these volume fractions can be calculated and scaled from the known inputs.
The first one would be more flexible, as it would allow units to override the calculations, but it would be a bit unsightly, requiring two terms called volume fraction. The opposite is true of the second.

**Screenshots**
Entered in-line above.

**SasView version (please complete the following information):**
 - Version: 5.0.6 release

**Operating system (please complete the following information):**
 - OS: Windows 10

**Additional context**
Just so that you can check my math, here are the values for volumes that I calculated using my inputs.

Volume core (radius) = 4/3*pi*200^3 = 3.35e7
Volume total (radius+thickness) = 4/3*pi*(200+100)^3 = 1.13e8
Volume shell (v_total-v_core) = 7.96e7

Rescaling of volume fraction for v_total = 1.13e8/7.96e7 = 1.42

Corrected volume fraction = 1.42*0.20 = 0.28

**Comment by gnsmith** [comment]
Here is the project file that I was using.
[vesicle_pqsq.json](https://github.com/SasView/sasview/files/14022784/vesicle_pqsq.json)

**Comment by murphyryanp** [comment]
@gnsmith: I was confused by this same issue a while ago. I think sasmodels is calculating the total (or excluded) volume fraction based on the shell volume fraciton. I was confused why the parameter "volfraction" is not explicitly called something distinct, like "shell_volfraction", to avoid confusion on whether the S(q) is indeed using the proper volfraction.

To visually confirm this is the case, I've plotted below a core-shell sphere model (orange) and vesicle model (blue), where the core_shell_sphere has core SLD = solvent SLD. I used the same parameters as you did in your vesicle example, and then plotted the results against a core-shell sphere with volume fraction 0.2 (results should be different) and 0.2839 (results should be identical).

Results with core_shell_volfraction = 0.2 and vesicle_volfraction = 0.2. These should be different in P(q) [top] and S(q) [bottom] and they are.
<img width="1298" alt="Screenshot 2024-01-26 at 4 56 14 AM" src="https://github.com/SasView/sasview/assets/30883207/113928b1-de7f-4ea9-86f5-3a040d696014">


Results with core_shell_volfraction = 0.2839 and vesicle_volfraction = 0.2. These should be identical in P(q) [top] and S(q) [bototm], and they visually appear to be the same.
<img width="1401" alt="Screenshot 2024-01-26 at 4 56 58 AM" src="https://github.com/SasView/sasview/assets/30883207/929efd4e-323d-4765-9913-a6473b435e23">

In short, I think the volfraction is being recalculated into the total volume fraction (behind the scenes) for purposes of the structure factor. I was still confused by the chosen parameter name!

**Comment by pkienzle** [comment]
The model and the docs are coming from sasmodels, so this is a sasmodels issue (see discussion in [sasmodels #232](https://github.com/SasView/sasmodels/issues/232)).

**Comment by pkienzle** [comment]
The description of the volfraction parameter in the model is "volume fraction of shell". Showing the parameter description in the GUI would help reduce the confusion. Maybe extend the description to say that it is uses the volume fraction of the enclosed volume when computing the structure factor.
